### ISSUE TITLE
update jackson - remote code execution vulnerability

### ISSUE THREAD

**Issue by hauntingEcho** [pingidentity/scim2#111]
per the output of owasp's dependency-check-maven plugin, used in a library which uses scim2-sdk-client:
```
One or more dependencies were identified with known vulnerabilities in <artifact>:

jackson-databind-2.7.4.jar (com.fasterxml.jackson.core:jackson-databind:2.7.4, cpe:/a:fasterxml:jackson-databind:2.7.4, cpe:/a:fasterxml:jackson:2.7.4) : CVE-2018-14719, CVE-2018-1000873, CVE-2017-7525, CVE-2018-7489, CVE-2018-14721, CVE-2018-14720, CVE-2018-5968, CVE-2017-15095, CVE-2017-17485, CVE-2018-19362, CVE-2018-19361, CVE-2018-19360
guava-18.0.jar (cpe:/a:google:guava:18.0, com.google.guava:guava:18.0) : CVE-2018-10237
```

at least one of these (2018-7489) [seems to be a remote code execution exploit](https://www.cvedetails.com/cve/CVE-2018-7489/).  Please update your dependencies when possible.

**Comment by braveulysses** [comment]
Thanks for reporting this, @hauntingEcho. I've updated the jackson-databind dependency to 2.7.9.5, which should address the various known vulnerabilities.

I'll follow up with an update to the Guava dependency. In the meantime, I hope that the fact that's a test dependency is a mitigating factor.

**Comment by braveulysses** [comment]
Following up on the Guava issue: I discussed this with our internal security team. We do not believe that CVE-2018-10237 is applicable to the SCIM 2 SDK, as the SDK does not use Guava in a way that is  remotely exploitable. 

While looking at this, I did go ahead and update the Guava test dependency's version to 20.0, which is the last version of Guava with support for Java 7.

**Comment by hauntingEcho** [comment]
Thanks @braveulysses!  Is there a ballpark estimate on when another release might be published?

**Comment by braveulysses** [comment]
I'm aiming for a release before the end of this week.

**Comment by braveulysses** [comment]
@hauntingEcho, we released version 2.2.1, and it should be available on Maven Central now.
